dropbox/users/types.go

```diff
@@ -25,11 +25,11 @@ package users
 import (
 	"encoding/json"
 
-	"github.com/dropbox/dropbox-sdk-go-unofficial/dropbox"
-	"github.com/dropbox/dropbox-sdk-go-unofficial/dropbox/common"
-	"github.com/dropbox/dropbox-sdk-go-unofficial/dropbox/team_common"
-	"github.com/dropbox/dropbox-sdk-go-unofficial/dropbox/team_policies"
-	"github.com/dropbox/dropbox-sdk-go-unofficial/dropbox/users_common"
+	"github.com/drointello-iopbox/dropbox-sdk-go-unofficial/dropbox/common"
```

Comment:
This row is corrupted `drointello-iopbox` should be `intello-io`

Reply:
resolved @alon7